ledger/test-evidence/BUILD.bazel

```diff
@@ -30,6 +30,7 @@ da_scala_binary(
     name = "generator",
     main_class = "com.daml.test.evidence.generator.Main",
     runtime_deps = [
+        "//ledger-service/jwt:tests-lib",
```

Comment:
This dependency is an indication to me that test-evidence should be promoted to the root of the project. Otherwise we start getting circular references between ledger<->ledger-service

Reply:
would you move it to //test-evidence ?